# In millimeters
float64 zone_mm
---
# Success if empty
string error
